packages/jss/src/index.js

```diff
@@ -7,7 +7,7 @@
  * @license MIT
  */
 import Jss from './Jss'
-import type StyleSheet from './StyleSheet'
```

Comment:
I wasn't sure whether this could be removed or not. It uses the same identifier (StyleSheet) so if it's to be retained I'll need to work out how to do this. I've not worked with flow types before